### ISSUE TITLE
How does anti-cheat work?

### ISSUE THREAD

**Issue by nagthanm** [fit-together-tv/ft-issue-report#12]

**Comment by bugless-it** [comment]
The anti-cheat actually works by comparing user answers, and highlights in red if the answer similarity is above a certain %. Can you show me the submission page for the above - are there actual answers for the questions?

**Comment by nagthanm** [comment]
> The anti-cheat actually works by comparing user answers, and highlights in red if the answer similarity is above a certain %. Can you show me the submission page for the above - are there actual answers for the questions?

![Anti Cheat 2](https://user-images.githubusercontent.com/87641533/129433246-92b30df1-f770-4f83-b946-5f1456d850b6.png)

**Comment by bugless-it** [comment]
> > The anti-cheat actually works by comparing user answers, and highlights in red if the answer similarity is above a certain %. Can you show me the submission page for the above - are there actual answers for the questions?
> 
> ![Anti Cheat 2](https://user-images.githubusercontent.com/87641533/129433246-92b30df1-f770-4f83-b946-5f1456d850b6.png)

can you share with me the URL?

**Comment by nagthanm** [comment]
https://app.chuongnau.com/me/question-form/30?currentFormMode=edit&currentTab=viewSubmission

Hi, here you are

**Comment by nagthanm** [comment]
![Anti Cheat 3](https://user-images.githubusercontent.com/87641533/129466053-5f85c664-787c-4e19-a648-9f782f02a19b.png)

https://app.chuongnau.com/me/question-form/36?currentFormMode=edit&currentTab=viewSubmission

It suppose to look like this, doesn't it?

**Comment by bugless-it** [comment]
@nagthanm yes it should look like that. Do you think it should be improved to be easier to understand?

**Comment by nagthanm** [comment]
From the first time I look at the anti-cheat, I do not under stand it at all.

You said: "The anti-cheat actually works by comparing user answers, and highlights in red if the answer similarity is above a certain %. "
I want to ask is the highlight in red showed the similarity between the wrong answers or it shows the similarity between both right and wrong answer?

Yeah, I think we should think how to make user can understand the function easier!!!

**Comment by nagthanm** [comment]
I understood the anti-cheat now. I have to look both at the result and the anti-cheat for the best evaluation.
I think the first time user might still have a bit of difficulty on how to figure it out.
I think we can add a ! or ? icon for explanation or I can make a instruction video. That is what I think for the solution.


> @nagthanm it works like a matrix - it compares a student's answer against all other students, and highlight the pair with the most similar answers. I understand it's a bit hard to understand as a user - do you have any suggestion on this?
